### DISCUSSION TITLE
How to View the Page History Version

### DISCUSSION THREAD

**Discussion by Marmot1024** [docmost/docmost#72] [Help / Questions] [answered]
Hello, I would like to ask why after deploying the project locally, after writing a document piece by piece, I am unable to view the history of the page. Is it necessary to press a button to save the current page in order to generate a version? Also, I hope you can reply to me, although this question might be pretty basic, haha.

**Comment by Philipinho** [comment]
Click on the three dots at the top right of the editor. There you have it.

**Reply by Marmot1024** [reply]
First of all, thank you for your reply. However, I'm not talking about not being able to find the entrance to view the historical versions. Instead, the document I wrote did not generate any versions. As shown in the following figure, I would like to know how to generate historical versions.
![20240707](https://github.com/docmost/docmost/assets/63297417/f63840a3-8619-41c5-809e-12c2c50ef8d4)

**Reply by Philipinho** [reply, marked as answer]
You cannot manually generate the page history. 
The page history generation happens every 10 minutes if there are sufficient changes.

**Reply by Marmot1024** [reply]
Alright, I have understood, thank you for your explanation😄

**Reply by Marmot1024** [reply]
Hello, may I inquire if you will support mind mapping plugins similar to Xmind in your document editing interface in the future?😄

**Reply by Philipinho** [reply]
I have plans to support MermaidJs first. I will consider Excalidraw and Draw.io.

**Reply by lephtHanded** [reply]
> I have plans to support MermaidJs first. I will consider Excalidraw and Draw.io.

I'd love excalidraw integration.

**Reply by ahmadnugroho-asp** [reply]
@Philipinho mine, the page history not generated after waiting for more than 2 days.
